src/config.template.cjs

```diff
@@ -167,144 +180,174 @@ module.exports = {
+	 * ago.
+	 * Doesn't exclude previously failed searches.
 	 * Examples:
-	 * "10min"
-	 * "2w"
-	 * "3 days"
+	 * "3 days".
+	 * "1 week".
+	 * "1 month".
 	 */
-	excludeRecentSearch: undefined,
+	excludeRecentSearch: "3 days",
 
 	/**
-	 * What action to take upon a match being found
-	 * Options: "save", "inject"
+	 * What action to take upon a match being found.
+	 * Options: "save", "inject".
 	 */
-	action: "save",
+	action: "inject",
 
 	/**
-	 * qBittorrent and Deluge specific
-	 * Whether to inject using the same labels/categories as the original torrent.
-	 * qBittorrent: This will apply the category's save path
-	 * Example: if you have a label/category called "Movies",
-	 * this will automatically inject cross-seeds to "Movies.cross-seed"
+	 * qBittorrent and Deluge specific.
+	 * Whether to inject using the same labels/categories as the original
+	 * torrent.
+	 * qBittorrent: This will apply the category's save path.
+	 * Example: if you have a label/category called "Movies", this will
+	 * automatically inject cross-seeds to "Movies.cross-seed".
 	 */
 	duplicateCategories: false,
 
 	/**
-	 * Run rss scans on a schedule. Format: https://github.com/vercel/ms
+	 * Run rss scans on a schedule.
 	 * Set to undefined or null to disable. Minimum of 10 minutes.
 	 * Examples:
-	 * "10min"
-	 * "2w"
-	 * "3 days"
+	 * "15 minutes".
+	 * "3 days".
+	 * "2 weeks".
 	 */
-	rssCadence: undefined,
+	rssCadence: "30 minutes",
 
 	/**
-	 * Run searches on a schedule. Format: https://github.com/vercel/ms
+	 * Run searches on a schedule.
 	 * Set to undefined or null to disable. Minimum of 1 day.
-	 * If you have RSS enabled, you won't need this to run often (2+ weeks recommended)
+	 * If you have RSS enabled, you won't need this to run often (2+ weeks
+	 * recommended)
 	 * Examples:
-	 * "10min"
-	 * "2w"
-	 * "3 days"
+	 * "1 day".
+	 * "3 days".
+	 * "2 weeks".
 	 */
-	searchCadence: undefined,
+	searchCadence: "1 day",
 
 	/**
 	 * Fail snatch requests that haven't responded after this long.
 	 * Set to null for an infinite timeout.
-	 * Format: https://github.com/vercel/ms
 	 * Examples:
-	 * "30sec"
-	 * "10s"
-	 * "1min"
-	 * null
+	 * "10 seconds".
+	 * "1 minute".
+	 * null.
 	 */
 	snatchTimeout: undefined,
 
 	/**
 	 * Fail search requests that haven't responded after this long.
 	 * Set to null for an infinite timeout.
-	 * Format: https://github.com/vercel/ms
 	 * Examples:
-	 * "30sec"
-	 * "10s"
-	 * "1min"
-	 * null
+	 * "30 seconds".
+	 * "1 minute".
+	 * null.
 	 */
 	searchTimeout: undefined,
 
 	/**
 	 * The number of searches to be done before it stops.
-	 * Combine this with "excludeRecentSearch" and "searchCadence" to smooth long-term API usage patterns.
-	 * Default is no limit.
+	 * Combine this with "excludeRecentSearch" and "searchCadence" to smooth
+	 * long-term API usage patterns.
+	 * Set to null for no limit.
 	 */
-	searchLimit: undefined,
+	searchLimit: 100,
```

Comment:
I still feel that if we are going to suggest this limit, perhaps a disclaimer clarifying this behavior for initial searches might be worth considering.

See previously resolved review comment. https://github.com/cross-seed/cross-seed/pull/619/files#r1536108533